File: CS/SelectedObjects.Module/Updater.cs
using System;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace SelectedObjects.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Contact Sam = ObjectSpace.CreateObject<Contact>();
            Sam.FirstName = "Sam";
            Sam.Save();
            Contact John = ObjectSpace.CreateObject<Contact>();
            John.FirstName = "John";
            John.Notes = "note";
            John.Save();
        }
    }
}

File: VB/SelectedObjects.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace SelectedObjects.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As ObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim Sam As Contact = ObjectSpace.CreateObject(Of Contact)()
			Sam.FirstName = "Sam"
			Sam.Save()
			Dim John As Contact = ObjectSpace.CreateObject(Of Contact)()
			John.FirstName = "John"
			John.Notes = "note"
			John.Save()
		End Sub
	End Class
End Namespace
